ui/src/tempVars/utils/replace.ts

```diff
@@ -28,12 +33,30 @@ export const replaceInterval = (
   return replaceAll(query, TEMP_VAR_INTERVAL, `${msPerPixel}ms`)
 }
 
+const TEMPLATES_SORTING_ORDER = {
```

Comment:
a comment may be useful to explain why there's a categorical ordering rather than a fixed absolute order here